packages/cli/demo/README.md

```diff
@@ -257,7 +257,7 @@ _Familiar Chat_ is an example application that provides a web app for
 interacting with your pet daemon.
 
 ```
-> endo install familiar-chat cat.js --powers SELF
+> endo install familiar-chat cat.js 8920 --powers SELF
```

Comment:
Wrong parameter order:
```suggestion
> endo install familiar-chat 8920 cat.js --powers SELF
```

Reply:
In light of this, perhaps we should add some parameter validation in the CLI for the name, port, and program path?

Reply:
Done, and changed `--name` and `--port` to required flags instead of a positional arguments.